### DISCUSSION TITLE
Version Dropdown

### DISCUSSION THREAD

**Discussion by JacobCoffee** [lepture/shibuya#40] [Q&A] [answered]
This is probably pretty dumb but I am not able to get the version dropdown to render outside of the caret icon. Did I fumble some config? I would like it to be fully visible

Example https://docs.advanced-alchemy.litestar.dev/latest/

<img width="271" alt="image" src="https://github.com/lepture/shibuya/assets/45884264/e8f3647d-fb0c-4a42-84ac-00be2c26db79">

**Comment by lepture** [comment, marked as answer]
<img width="329" alt="image" src="https://github.com/lepture/shibuya/assets/290496/92089f9f-31a1-4568-a4f0-561ab0788ad9">

It is caused by your CSS. I suggest that you remove tailwindcss in [litestar-sphinx-theme](https://github.com/litestar-org/litestar-sphinx-theme) to avoid the problem.

**Reply by JacobCoffee** [reply]
Thanks! I think this will be a little harder because https://litestar.dev/ depends on this as well 🙁
